src/ConnectedMode/Install/ImportBeforeInstallTrigger.cs

```diff
@@ -78,8 +78,8 @@ public void Dispose()
         {
             if (!disposed)
             {
-                this.activeSolutionBoundTracker.PreSolutionBindingChanged -= OnPreSolutionBindingChanged;
-                this.activeSolutionBoundTracker.PreSolutionBindingUpdated -= OnPreSolutionBindingUpdated;
+                activeSolutionBoundTracker.PreSolutionBindingChanged -= OnPreSolutionBindingChanged;
```

Comment:
I didn't explicitly make these changes, so presumably VS did it automatically.

I'll check my VS settings if it happens again. We don't really want a whole load of unplanned cosmetic changes being applied automatically and obscuring the real changes in a PR.